indexing-service/src/main/java/org/apache/druid/indexing/overlord/ZkWorker.java

```diff
@@ -167,6 +167,29 @@ private int getCurrParallelIndexCapacityUsed(Map<String, TaskAnnouncement> tasks
     return currParallelIndexCapacityUsed;
   }
 
+  @JsonProperty("currTypeSpecificCapacityUsed")
+  public Map<String, Integer> getCurrTypeSpecificCapacityUsed()
```

Comment:
I thought we had deprecated Zk based runner in favour of http. 
@kfaraz Does this change still make sense ?

Reply:
Yes, ZK-based task runner is deprecated and we should not support the new feature with ZK.